Formula/f/faircamp.rb

```diff
@@ -0,0 +1,30 @@
+class Faircamp < Formula
+  desc "Static site generator for audio producers"
+  homepage "https://codeberg.org/simonrepp/faircamp"
+  url "https://codeberg.org/simonrepp/faircamp/archive/0.9.2.tar.gz"
+  sha256 "15c826e22d8297223ae6c591d14ff79226682ef4db3911a1f349d639406295b7"
+  license "AGPL-3.0-or-later"
+
+  depends_on "ffmpeg" => :build
+  depends_on "opus" => :build
+  depends_on "pkg-config" => :build
+  depends_on "rust" => :build
+  depends_on :macos # Linux builds aren't working at present
+  depends_on "vips"
+
+  def install
+    # libvips is a runtime dependency as it only ships a dynamic
+    # library with brew, its not discovered by default by cargo:
```

Comment:
is there any upstream issue for this?

Reply:
For ‘libvips’? Is it expected that it should have both? Wasn’t sure if it was an issue or just “how things are”? Is it policy that `brew` distributed libs should have static and dynamic variants included?

Reply:
Reworded to hopefully better explain the situation [here](https://github.com/Homebrew/homebrew-core/pull/153696/commits/7b25f6e216b7ce4bccc5525afc8f5808302fa75e).